code/controllers/subsystem/ticker.dm

```diff
@@ -554,18 +554,22 @@ SUBSYSTEM_DEF(ticker)
 	// Declare ship objectives
 	to_chat(world, "<br><FONT size=3><B>The ship objectives were:</B></FONT>")
 	var/count = 1
-	var/redtext = 0
+	var/redtext = FALSE
+	if(!locate(/datum/objective/ftl/gohome) in get_ship_objectives())
+		redtext = TRUE
 	for(var/datum/objective/objective in get_ship_objectives())
-		if(objective.check_completion() && !objective.failed)
-			to_chat(world, "<br><b>Objective #[count]</b>: [objective.explanation_text] <span class='greenannounce'>Success!</span>")
+		if(objective.type == /datum/objective/ftl/gohome && !objective.check_completion())
+			to_chat(world, "<br><b>Objective #[count]</b>: [objective.explanation_text] <span class='boldannounce'>Failed.</span>")
+			redtext = TRUE
+		else if(objective.check_completion() && !objective.failed)
+			to_chat(world, "<br><b>Objective #[count]</b>: [objective.explanation_text] <span class='greenannounce'>Completed!</span>")
 		else
-			to_chat(world, "<br><b>Objective #[count]</b>: [objective.explanation_text] <span class='boldannounce'>Fail.</span>")
-			redtext = 1
+			to_chat(world, "<br><b>Objective #[count]</b>: [objective.explanation_text] <span class='boldannounce'>Failed.</span>")
 		count++
-	if(redtext)
+	if(!redtext)
```

Comment:
??

Reply:
Yeah, big oops. Resolved in https://github.com/FTL13/FTL13/pull/3654/commits/c0fcc0c5e1228f7dea7fc3332dbb3cc27f70f2aa